### ISSUE TITLE
Support svg.gz in outputs viewer

### ISSUE THREAD

**Issue by bloodearnest** [opensafely-core/job-server#2099]
Users like .svg files as an export format. However, they can get very large, too large to release.

.svgz  or .svg.gz formats are much smaller, but cannot currently be viewed in the outputs viewer.
It would be could to add support for them.

In concert, it would be good to update the [releasing docs](https://docs.opensafely.org/releasing-files/#2-requesting-release-of-outputs-from-the-server) to encourage compresses svg

**Comment by sebbacon** [comment]
Just to say, we have talked before about requiring images to be bitmaps, because SVGs can more easily leak identifiable information. Did a decision get made at some point that this is an acceptable risk, do you know?

**Comment by bloodearnest** [comment]
I don't think so. Surely the same problem applies to HTML?

The reviewers look at the rendered output only in both cases.

**Comment by sebbacon** [comment]
This is a fair question. You could argue SVG is different because as I understand it many libraries include loads of granular data by default even in aggregated views. But I don't really know, and I don't have a view on the answer.

The reason for asking the question is that I know there _have_ been questions like this, and it makes me think we should probably be careful to make (and capture) decisions about file types with the output-release team or experts (like @LFISHER7).

**Comment by LFISHER7** [comment]
I don't think we explicitly made a call against SVGs. We ask for the underlying data for any figures when output checking but there's a certain amount of trust that that is what is being plotted. We can certainly encourage bitmaps as the preferred format in the docs. A big part of this is allowing the flexibility to meet the various requirements of journals.
